app/models/system_notification.rb

```diff
@@ -13,7 +13,7 @@ class SystemNotification < ApplicationRecord
                                            }
   #  ignoring: :accents
 
-  has_many :user_system_notifications
+  has_many :user_system_notifications, dependent: :destroy
```

Comment:
Hm... Not sure if we have a case where we delete SystemNotification, but if we do and this is on Free Scinote, it will call destroy for each `user_system_notification` record and this can take quite a long time. 
Should we be concerned?